rust/interpreter/src/evaluate.rs

```diff
@@ -112,6 +133,37 @@ pub fn evaluate(emit: &EmitResult) -> Result<JSValue, EvalError> {
                 continue;
             }
 
+            Opcode::GetGName => {
+                let atom = emit.read_atom(pc + 1);
+                stack.push(global.borrow().get(atom));
+            }
+
+            Opcode::GImplicitThis => {
+                stack.push(JSValue::Object(global.clone()));
```

Comment:
This should push `JSValue::Undefined`; see [the documentation of GImplicitThis](https://searchfox.org/mozilla-central/rev/ad6234148892bc29bf08d736604ac71925138040/js/src/vm/Opcodes.h#1845-1846).